protocols/imap/src/main/java/org/apache/james/imap/decode/main/DefaultImapDecoder.java

```diff
@@ -121,7 +122,9 @@ private ImapMessage unknownCommand(String tag, ImapSession session) {
     private ImapMessage decodeCommandNamed(ImapRequestLineReader request, String tag, String commandName, ImapSession session) {
         ImapMessage message;
         if (session.getLog().isDebugEnabled()) {
-            session.getLog().debug("Got <command>: " + commandName);
+            if (!ImapConstants.UID_COMMAND_NAME.equals(commandName)) {
```

Comment:
Doing it in an other way would require me to duplicate Netty buffer for connection...

I can remove the IF (log more) if you prefer